builtin/logical/pki/cert_util.go

```diff
@@ -258,6 +258,10 @@ func fetchCertBySerial(ctx context.Context, req *logical.Request, prefix, serial
 // If one does not pass, it is returned in the string argument.
 func validateNames(req *logical.Request, names []string, role *roleEntry) string {
 	for _, name := range names {
+		if name == "" {
```

Comment:
I don't think this should happen... Don't see a reason why we would want empty names in that list.

Reply:
Removed.